pybay/pybay_old
# Testing the Speakers List Page

*Please run this command to install the Model mommy: pip install model_mommy
- Test the view
- Make sure it only displays the accepted users
- Make sure accepted speakers has a photo, do not display speakers without a photo in the speakers list page
- Check the additional speaker
